languages/js/INSTALL.md

```diff
@@ -0,0 +1,11 @@
+# WASM SDK Instructions
+
+## Steps
+
+1. Run the wasm build script: `./sdk/crates/bitwarden-wasm/build.sh`
```

Comment:
Should we link to the wasm read me here? 

https://github.com/bitwarden/sdk/blob/main/crates/bitwarden-wasm/README.md

It requires installing some extra stuff to run that build script.

Reply:
Resolved in: https://github.com/bitwarden/sdk/pull/1044/commits/7cb94c737f34a5f5fbb47420ff08ad2fd9802e0c